ringo/lib/imexport.py

```diff
@@ -160,28 +160,16 @@ class Exporter(object):
     export related items if configured correct. In this case the
     Exporter will return a list of nested dictionaries.
 
-    You can configure which will be exported on each item by either
-    using the `fields` parameter. If no fields are provided all fields
-    excluding the relations will be exported. The order of the
-    configured fields will determine the order of the fields in the
-    export (If supported e.g CSV).
-
-    A more detailed option to configur the content of the export you can
-    provide a ExportConfiguration to the exporter.
+    As a detailed configuration of the content of the export you can
+    provide an ExportConfiguration to the exporter.
 
     On default the exported items will be `serialized`. This means
     that each value is converted into a export specific format. E.g
     dates are converted into ISO8601 notation in the JSONExporter.
-    If set to false the the values are real python values.
-
-    Using the `relations` parameter is deprecated. It can be used as a
-    shortcut to add ORM relation of the item into the export.  Exported
-    relations will be the id of the linked items. In connection with the
-    serialized parameter the string representation of the linked items
-    are exported.
+    If set to false the values are real python values.
     """
 
-    def __init__(self, clazz, fields=None, serialized=True, relations=False, config=None):
```

Comment:
See my last comment in #86. I think it is OK to remove those options as we can do the same using an export config.

Reply:
Ok, thanks.

I addressed your request for changes in 698122b.